### PR TITLE
Data dev

### PR DESCRIPTION
**Pull request by mstfdkmn**
I added new files and also updated the initial files based on your inputs. Though there are two files that will be written later (Yoda, python client), i put them here on behalf of integrity. I know some pictures in files look not good but for a first step i keep how they seem. I will update them in accordance with your feedback. Thank you!

### CODE REVIEW

File: source/data/tier1_data_architecture.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.. _tier1_data_architecture:
+
+Tier-1 Data Platform (iRODS) Architecture
+=========================================
+
+VSC, as an integrated high-performance research computing environment, provides services to government, industry, and researchers. For this purpose it has developed a differentiated infrastructure (Tier-1 and Tier-2 level) that is available to use. On top of the Tier-1 a Data component (iRODS) has been initialized to manage research data.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:6]:
```suggestion
The Tier-1 Data platform is based on the iRODS open source software. A high level architecture schema can be seen n the following image: (move architecture/general_overview.png here)
```

Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:6]:
```suggestion
The VSC Tier-1 a Data component is based on the open source software iRODS. The following image shows the high level archirecture of the platform (move architecture/general_overview.png image here)

File: source/data/tier1_data_architecture.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+An iRODS zone represent the deployment of the system. In the architecture of the current deployment “kuleuven_tier1_pilot” is the only zone that serves for VSC. Inside the zone there are two physical locations as well as replicas of logical representation. We have tier1-p-irods-posix-{1,3} in Heverlee for main storage and we have tier1-p-irods-posix-{2,4} in Leuven for replicas.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:8]:
```suggestion
The current deployment is based in a unique iRODS zone (“kuleuven_tier1_pilot”) with a single iCAT database configured on High Availability. There are three distributed storage resources: 2 POSIX based systems and 1 Ceph Object Storage system). 1 of the POSIX systems and the Ceph based storage are physically installed at the KU Leuven Heverlee datacenter while the other POSIX system is installed at the KU Leuven datacenter located in Leuven center and act as a replica of the POSIX system located in Heverlee. 
That means that for every object registered in the Data platform 2 copies will be automatically stored (one in Heverlee and one Leuven). 
The third storage system will be used for the moment to store a third copy of some selected data objects. 
```

File: source/data/tier1_data_architecture.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+.. image:: architecture/resource.png
+
+As part of the kuleuven_tier1_pilot zone, to manage the research data there are rule engine packages and iRODS metadata catalog-iCAT. When a user accesses iRODS, a communication inside the irods happens. The user’s any request is forwarded to the appropriate iRODS Server.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:13]: Too technical information. Better to leave it at a more high level

```suggestion

```

File: source/data/tier1_data_architecture.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+An iRODS zone represent the deployment of the system. In the architecture of the current deployment “kuleuven_tier1_pilot” is the only zone that serves for VSC. Inside the zone there are two physical locations as well as replicas of logical representation. We have tier1-p-irods-posix-{1,3} in Heverlee for main storage and we have tier1-p-irods-posix-{2,4} in Leuven for replicas.
+
+
+.. image:: architecture/resource.png
+
+As part of the kuleuven_tier1_pilot zone, to manage the research data there are rule engine packages and iRODS metadata catalog-iCAT. When a user accesses iRODS, a communication inside the irods happens. The user’s any request is forwarded to the appropriate iRODS Server. 
+The Rule Engine at that location then verifies whether the desired operations can be executed, translates the operations into the protocol required by that type of storage system, and passes the result of the operations back to the iRODS client. Any state information that is generated is registered into the iCAT metadata catalog.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:14]: Again, too technical, not necessary
```suggestion

```

File: source/data/tier1_data_architecture.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+A user who wants to access iRODS can use a local computer and/or HPC servers. This accessions are done through user clients which are for now consists of a default client iCommands, a portal client, WebDAV clients and a Python Client.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:16]:
```suggestion
A user can access iRODS from a local computer and/or the VSC Tier-1 and Tier-2 systems using different types of user clients.  At this moment they are available:  iCommands, a portal client , WebDAV clients and a Python Client.
```

File: source/data/tier1_data_architecture.rst [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+iCommands is an utility that give users a command-line interface to operate on data in iRODS. 
+
+The portal client-YODA is a user friendly web based application to provide researchers and their partners with a workspace and an archive that enables them to collaborate, deposit and preserve research data.
+
+With the aid of WebDAV protocol, a Drag and Drop Access to iRODS is ensured by means of some apps/tools (e.g. WebDAV mapping, Cyberduck and WinSCP) that enable data transfer.
+
+PRC is a Python Client API to establish a secure connection to iRODS in order to manage the research data.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:24]:
```suggestion
PRC is a Python Client API to establish a secure connection to iRODS an be able to interoperate with iRODS from python programs.
```

File: source/data/tier1_data_architecture.rst [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+
+.. image:: architecture/general_overview.png
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:27]: Move this image at the beginning

File: source/data/tier1_data_architecture.rst [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+A general overview of kuleuven_tier1_pilot is simulated with the diagram at the picture above.
```
Comment (ingridbr) [inline source/data/tier1_data_architecture.rst:29]:
```suggestion

```

File: source/data/tier1_data_architecture.rst (hunk 7 above)
Comment (mstfdkmn) [reply source/data/tier1_data_architecture.rst:26]: ok

**Comment by mstfdkmn** [comment]
Thank you for your inputs

**Pull request merged**